src/utils/sortTableValues.js

```diff
@@ -0,0 +1,20 @@
+const sortTableValues = (key, order = "asc") => {
```

Comment:
The file can be named `sort.js` or `sortUtils.js`

Reply:
Changed to sort.js